llvm/lib/Target/X86/X86SelectionDAGInfo.cpp

```diff
@@ -146,8 +154,8 @@ SDValue X86SelectionDAGInfo::EmitTargetCodeForMemset(
                     DAG.getNode(ISD::ADD, dl, AddrVT, Dst,
                                 DAG.getConstant(Offset, dl, AddrVT)),
                     Val, DAG.getConstant(BytesLeft, dl, SizeVT), Alignment,
-                    isVolatile, AlwaysInline,
-                    /* CI */ nullptr, DstPtrInfo.getWithOffset(Offset)));
+                    isVolatile, /*AlwaysInline*/ true, /* CI */ nullptr,
```

Comment:
Why have you overriden AlwaysInline to true?

Reply:
Not anymore.

Reply:
Though I did this to match memcpy, which does do this, and I don't think we should do a call when we know that this is for 7 bytes at most.

Reply:
Anyway, I undid that already.